lib/prod-server/app.js

```diff
@@ -53,15 +53,21 @@ module.exports = ({
   }
   server.use((req, res) => {
     const url = req.url.replace(baseMatch, '/')
-    runner(url).then(({ result, redirect }) => {
+    runner(url).then(({ result, redirect, options = {} }) => {
```

Comment:
I don't like `options` 

`{ result, redirect, status, headers, body }` would be better I think.

If you want the "more complete" behaviour you can use the three new keys (`status, headers, body`).

Reply:
How would `body` differ from `result`?

Reply:
It wouldn't at all, except that it's more in line with an object that represents a HTTP response.